Casks/x/xliffeditor.rb

```diff
@@ -0,0 +1,25 @@
+cask "xliffeditor" do
```

Comment:
I think this should be `xliff-editor`. The filename needs to be updated as well.
```suggestion
cask "xliff-editor" do
```